modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,20 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option allows you to split the current container block element if the enter key is pressed inside an empty inner block element, and specify the type of element created.
```

Comment:
```suggestion
This option allows you to split the current container block element and specify the type of element created if the enter key is pressed inside an empty inner block element.
```

I moved the two option-related phrases together.

The original has an unstated but easily inferred ‘then’ adverb. That is, it’s very easy to read the original as meaning

> This option allows you to split the current container block
> element if the enter key is pressed inside an empty inner block
> element, and _then_ specify the type of element created.


This reading — which makes the inferred causal relationship explicit — turns the option into something that acts as follows:

To use `end_container_on_empty_block` to split the current container

1. put the insertion point inside an empty inner block element

then

2. press *Enter*

then

3. specify the type of element to be created.

Given the option is set with a pre-chosen value, this inferred causal relationship is incorrect.

And moving the phrase makes this inference much harder to make.